.circleci/config.yml

```diff
@@ -216,6 +249,7 @@ jobs:
             cd telemetry-airflow
             bash bin/test-parse
   verify-dags-up-to-date:
+    # todo: remove this step once we rely on generate-dags completely
```

Comment:
This step can be removed once we copy DAGs from `generated-sql` instead of from the `main` branch